applications/openshift/registry/ocp_allowed_registries_for_import/rule.yml

```diff
@@ -0,0 +1,22 @@
+prodtype: ocp4
+
+title: "Allowed registries for import are configured"
+
+description: |-
+    The configuration <tt>allowedRegistriesForImport</tt> limits the container
+    image registries from which normal users may import images. This is important
+    to control, as a user who can stand up a malicious registry can then import
+    content which claims to include the SHAs of legimitate content layers.
+
+rationale: |-
+    Allowed registries for import should be specified to limit the registries 
+    from which users may import images.
+
+identifiers:
+    cce@ocp4: 84210-1
```

Comment:
Placeholder for now

Reply:
Let's remove this for now, since this is making the build break

Reply:
removed and rebased